[RM Ch01] Restructure to fit GSMA format (#1750)
cntt-n/CNTT: doc/ref_model/chapters/chapter01.md

[<< Back](../../ref_model)
# 1. Introduction

## Table of Contents
* [1.1 Overview](#1.1)
* [1.2 Scope](#1.2)
* [1.3 Principles](#1.3)
* [1.4 Definitions/Terminology](#1.4)
* [1.5 Abbreviations](#1.5)
* [1.6 References](#1.6)
* [1.7 Conventions](#1.7)

<a name="1.1"></a>
## 1.1 Overview

> The Reference Model (RM) specifies the infrastructure abstraction and the exposure of a set of capabilities, resources, and interfaces to workloads. The aim of the Reference Model is to be virtualisation technology agnostic (VM-based and container-based) and act as a "catalogue" of the exposed infrastructure capabilities, resources, and interfaces needed to develop the workloads. 

Reviewer: There is some repetition in the paragraph. 
```suggestion
The Reference Model (RM) specifies a virtualisation technology agnostic (VM-based and container-based) cloud infrastructure abstraction and acts as a "catalogue" of the exposed infrastructure capabilities, resources, and interfaces required by the workloads. 
```

- **Appendix A - Guidelines For Application Vendors**: More details related to how the applications will interface with the cloud infrastructure through APIs (including Cloud Infrastructure Manager and CaaS).
  > **Audience**: This chapter is written for architects, developers and others who need to deploy infrastructure or develop applications.

<a name="1.2"></a>
## 1.2 Scope

> This document focuses on the documenting the higher level concepts that are needed to identify **Reference Model**. **Figure 1-1** below highlights its scope in more details.

Reviewer: Concepts of what?
```suggestion
This **Reference Model** document focuses on identifying the abstractions, and associated concepts, that are needed to represent the cloud infrastructure. **Figure 1-1** below highlights its scope in more details.
```

<p align="center"><img src="../figures/ch01_scope.png" alt="scope" title="Scope" width="100%"/></p>
<p align="center"><b>Figure 1-1:</b> Scope of Reference Model</p>

This document specifies:

> - **Cloud Infrastructure abstraction**: in context with how it interacts with the other components required to build a complete system that supports workloads deployed in Virtual Machines (VNF) or containers (CNF).

Reviewer: ```suggestion
- **Cloud Infrastructure abstraction**: in context with how it interacts with the other components required to build a complete cloud system that supports workloads deployed in Virtual Machines (VM) or containers. Network function workloads that are deployed on virtual machines and containers are referred to as virtual network functions (VNF) and containerised network functions (CNF), respectively; please note that it is now more common to refer CNFs as cloud native network functions. 
```

>   - **Cloud Infrastructure metrics & capabilities**: A set of metrics and capabilities for the cloud infrastructure which workloads require to perform telco scale network functions.

Reviewer: Suggest capabilities before metrics
```suggestion
  - **Cloud Infrastructure capabilities & metrics**: A set of cloud infrastructure capabilities and metrics required to perform telco scale network functions and satisfy their performance criterion.
```

>   - **Infrastructure profiles catalogue**: A catalogue of standard profiles needed in order to abstract the infrastructure from workloads. With a limited and well-defined set of profiles with well understood characteristics, workload compatibility and performance predictability can be achieved.

Reviewer: What is a profile? 
```suggestion
  - **Infrastructure profiles catalogue**: A catalogue of standard infrastructure software and hardware configurations, referred to as profiles; these profiles abstract the infrastructure for the workloads. Only a few profiles, with well-defined characteristics, can meet the operational and performance requirements of all workloads.
```

> - Cloud Infrastructure Software and Hardware profiling

Reviewer: ```suggestion
- Cloud Infrastructure Software and Hardware profiles:
```

>   - **Cloud Infrastructure software profiles and configurations**: These are software profiles and configurations that map directly to the infrastructure profiles within the infrastructure profiles catalogue.

Reviewer: profiles are configurations

>   - **Cloud Infrastructure hardware profiles and configurations**: These are hardware profiles and configurations which are suitable for the defined cloud infrastructure software profiles & configurations.

Reviewer: similar comment as to software profiles

- Conformance and verification

>   - **Conformance programs**: This defines the requirements for certification and validation programs for both workloads and cloud infrastructure.

Reviewer: since programs plural; also certification is loaded term; primary focus cloud infrastructure and also workload conformance is highly limited in scope
```suggestion
  - **Conformance programs**: These define the requirements for verification and validation programs for both the cloud infrastructure and workloads.
```

>   - **Test framework**: Provide test suites to allow conformance, certification, and verification of workloads and cloud infrastructure against the defined set of profiles.

Reviewer: conformance is defined as verification and validation (suggested) and originally as certification and validation -- so why have these terms after conformance. Also, conformance is not only against profiles. Does the test frame work provide the suites or a platform for the test suites to be executed against the system under test. The suggestion that follows doesn't address this last point.
```suggestion
  - **Test framework**: Provides test suites to allow conformance of cloud infrastructure and workloads.
```

<a name="1.3"></a>
## 1.3 Principles

> The Reference Model specification conform with the overall principles defined in the preface chapter [Cloud iNfrastructure Telco Taskforce](../../tech#2.0).

Reviewer: ```suggestion
The Reference Model specifications conform with the overall principles defined in the preface chapter [Cloud iNfrastructure Telco Taskforce](../../tech#2.0).
```

<a name="1.4"></a>
## 1.4 Definitions/Terminology

> To help guide the reader, a [CNTT Terminology](../../tech/glossary.md) section in [Cloud iNfrastructure Telco Taskforce] provides an introduction to the main terms used within this document and throughout the project in general. These definitions are, with a few exceptions, based on the [ETSI GR NFV 003 V1.5.1 (2020-01)](https://www.etsi.org/deliver/etsi_gr/NFV/001_099/003/01.05.01_60/gr_NFV003v010501p.pdf) definitions.  In a few cases, they have been modified to avoid deployment technology dependencies only when it seems necessary to avoid confusion.

Reviewer: "section in section in [Cloud iNfrastructure Telco Taskforce] provides .." -- maybe the intent was to include a reference but even then should it be CNTT documents?
last sentence two "avoid"
Aren't some of the definitions from cloud native foundation?

Author: @pgoyal01, we have to think how we will incorporate "Cloud iNfrastructure Telco Taskforce" tech document into RM as Chapter 0, as agreed between Rabi and Tom.  Then we will clean this up.  Topic for the RM Meeting.

<a name="1.5"></a>

> ## 1.5	Abbreviations

Reviewer: Should we have a document for acronyms? Not sure why PRD is included here

| Term           | Description                                                                                                                |
|----------------|----------------------------------------------------------------------------------------------------------------------------|
| CNTT           | Cloud iNfrastructure Telco Taskforce  |  
| PRD            | Permanent Reference Document |
| RM             | Reference Model |

<a name="1.6"></a>

> ## 1.6	References 

Reviewer: References are usually the last section

Author: @pgoyal01, I have followed the GSMA format and sequence of sections here.  I propose to stick to their ways..
